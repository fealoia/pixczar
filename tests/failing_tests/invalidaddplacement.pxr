Void main() {
	Pix test = new Pix();
	Frame[] frames = new Frame[1];
	frames[0].addPlacement(test);
	//add pix instead of placement
}