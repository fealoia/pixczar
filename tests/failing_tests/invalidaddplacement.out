Fatal error: exception Failure("illegal object function call frames[0].addPlacement(test);
")
